src/h5web/App.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+import MockProvider from './providers/mock/MockProvider';
+
+const DOMAIN = 'bsa_002_000-integrate-sub';
+
+test('renders with root group selected', async () => {
+  const { findByText, findByRole } = render(
+    <MockProvider domain={DOMAIN}>
+      <App />
+    </MockProvider>
+  );
+
+  const noVisText = await findByText('Nothing to visualize');
+  expect(noVisText).toBeInTheDocument();
+
+  const title = await findByRole('heading', { name: DOMAIN });
+  expect(title).toBeInTheDocument();
```

Comment:
I've changed the breadcrumbs element to be a heading so I can query it in the tests.

Reply:
What's you're reasoning? Semantically, a `header` element would have to contain a heading element (i.e. `h1`) anyway, so I went for the simplest.

Reply:
My reasoning was the change from the regular `div` to a `header` instead of `h1` to avoid the overwritting of the styling of `h1` in the CSS. But it is fine like you did.